capstone/capapi/filters.py

```diff
@@ -423,7 +444,21 @@ class NestedFTSFilter(BaseSearchFilterBackend):
     ]
 
     separate_queries_per_field = True
+    combine_with_cite = False
     matching = MATCHING_OPTION_MUST
+    search_key = 'query_values'
+
+    def get_search_query_params(self, request, view):
+        params = super().get_search_query_params(request)
+
+        if self.combine_with_cite and params:
+            # Add params from multinestedfilteringbackend. View not needed.
+            cite_params = MultiNestedFilteringFilterBackend().get_filter_query_params(request, view, called_from_multi=True)
```

Comment:
manipulate inbound requests first and add a `opinion_` prefix to each relevant feature